src/components/pages/Friends/Friends.js

```diff
@@ -1,11 +1,61 @@
 import React from 'react';
+import smashRequests from '../../../helpers/data/smashRequests';
+import authRequests from '../../../helpers/data/authRequests';
+import FriendItem from '../../FriendItem/FriendItem';
+
 import './Friends.scss';
 
 class Friends extends React.Component {
+  state = {
+    users: [],
```

Comment:
since you are breaking your users up into 3 separate arrays i don't think there is any reason for you to hold the full array in memory.  You can always generate it if you need it by doing something like:

```
const {pendingFriends, potentialFriends, currentFriends} = this.state;
const users = [...pendingFriends, ...potentialFriends, ...currentFriends];
```

Reply:
Fixed in https://github.com/MCrank/react-nutshell/pull/13/commits/3c56bc4e76dc5e8953726da354218a30ac961889

Reply:
Fixed in https://github.com/MCrank/react-nutshell/pull/13/commits/3c56bc4e76dc5e8953726da354218a30ac961889